src/lang/en.json

```diff
@@ -859,5 +859,13 @@
     "successEnabled": "Enabled Successfully.",
     "tagNotFound": "Tag not found.",
     "foundChromiumVersion": "Found Chromium/Chrome. Version: {0}",
+    "Remote Browsers": "Remote Browsers",
+    "Remote Browser": "Remote Browser",
+    "Add a Remote Browser": "Add a Remote Browser",
+    "Remote Browser not found!": "Remote Browser not found!",
+    "remoteBrowsersDescription": "Remote Browsers are an alternative to running Chromium locally. Setup with a service like browserless.io or connect to your self-hosted container: https://hub.docker.com/r/browserless/chrome",
```

Comment:
`https://hub.docker.com/r/browserless/chrome` is no need to be here. It can be convert to `{0}` and also should be clickable.